openlcs/libs/kojiconnector.py

```diff
@@ -483,3 +489,51 @@ def download_source(self, build):
             err_msg = f'Failed to download source. Reason: {err}'
             raise RuntimeError(err_msg) from None
         return temp_dir
+
+    def get_module_mapping_srpms(self, module_nvr):
+        """
+        Return module SRPM NVRs.
+        """
+        build = self.get_build(module_nvr)
+        tag = build.get('extra').get('typeinfo').get('module').get(
+                'content_koji_tag')
+        try:
+            builds = self.list_tagged(tag)
+        except RuntimeError as err_msg:
+            err_msg = f'Failed to get the module {module_nvr} SRPM nvrs'
+            raise RuntimeError(err_msg) from None
+        package_nvrs = []
+        for build in builds:
+            package_nvrs.append(build.get('nvr'))
+        return package_nvrs
+
+    def get_module_component(self, module_nvr):
+        """
+        Return module component itself.
+        """
+        nvr = koji.parse_NVR(module_nvr)
+        module_component = [get_component_flat(nvr, 'RPMMOD')]
```

Comment:
Why a list is used?

Reply:
It likes the container/components structure:
{
RPM: []
OCI: []
}.

Reply:
If I remember correctly, the reason is to unify the process of getting the component_data for create_component(compnent_data) function. I have a look at the packages/mixin.py, it should create the parent_component first in save_group_component first, then create the child components in build_component_node. If the OCI:[{}] or RPMMOD:[{}] will be changed to OCI:{}, the related functions above should be updated.  
If you think the structure should be updated, that's ok for me to update the related codes.

Reply:
Updated.